numba_cuda/numba/cuda/cudadrv/driver.py

```diff
@@ -2735,30 +2764,24 @@ def new(
         additional_flags=None,
     ):
         driver_ver = driver.get_version()
-        if config.CUDA_ENABLE_MINOR_VERSION_COMPATIBILITY and driver_ver >= (
-            12,
-            0,
-        ):
-            raise ValueError("Use CUDA_ENABLE_PYNVJITLINK for CUDA >= 12.0 MVC")
-        if config.CUDA_ENABLE_PYNVJITLINK and driver_ver < (12, 0):
-            raise ValueError("Enabling pynvjitlink requires CUDA 12.")
-        if config.CUDA_ENABLE_PYNVJITLINK:
-            linker = PyNvJitLinker
-
-        elif config.CUDA_ENABLE_MINOR_VERSION_COMPATIBILITY:
-            linker = MVCLinker
+        if driver_ver <= (12, 0):
```

Comment:
Should this be `<=` or `<`? Did something change in the drivers for 12.1? Can we explicitly use `< (12, 1)` if that is the case?

Reply:
Bad local merge, thanks 🚀 
```suggestion
        if driver_ver < (12, 0):
```